internal/karma_test.bzl

```diff
@@ -89,3 +93,15 @@ karma_test = rule(
     },
     test = True,
 )
+
+def karma_test_macro(name, tags, **kwargs):
+  """Enable watch mode for karma_test rules.
+  """
+  # This tag tells ibazel that all karma_test targets are
+  # compatible with notify mode, so that when the target
+  # is executed with `ibazel run :foo`, ibazel will write
+  # a notification on stdin to the karma process rather than
+  # restart it after each re-build.
+  # https://github.com/bazelbuild/bazel-watcher/blob/d8caa87266a6fda7469d2d56dab607c18067e615/ibazel/ibazel.go#L265
+  tags = tags + ["IBAZEL_MAGIC_TAG"]
```

Comment:
I'm a bit surprised by this working. I had that as a placeholder value until I looked up the internal one. When I looked it up I only updated it in the description not the body. Yeah, it could use some more tests. Since this is a divergence from the internal tag style what would you like to do (ibazel_notify_changes)?

https://github.com/bazelbuild/bazel-watcher/commit/785235d1d5385a60c5a16fa002c9b8c1e943e36e